service/storage_service.cc

```diff
@@ -3554,5 +3554,16 @@ db::schema_features storage_service::cluster_schema_features() const {
     return f;
 }
 
+future<bool> storage_service::allow_cleanup(sstring keyspace) {
```

Comment:
Predicates should not read as imperatives, so I suggest e.g. "is_cleanup_allowed"

Reply:
Done.